docker-bake.hcl

```diff
@@ -1,41 +1,25 @@
 variable "REPO" {
   default = "tonistiigi/binfmt"
 }
-variable "TAG" {
-  default = ""
-}
-variable "LATEST" {
-  default = "false"
-}
 variable "QEMU_REPO" {
   default = ""
 }
 variable "QEMU_VERSION" {
   default = ""
 }
 
+// Special target: https://github.com/crazy-max/ghaction-docker-meta#bake-definition
+target "ghaction-docker-meta" {
```

Comment:
let's not call this with cryptic name.  "base", "_meta", "meta-helper" etc.

Reply:
That's currently [statically defined in the action](https://github.com/crazy-max/ghaction-docker-meta/blob/ae431178c17085afd420fdf7ca88b37bd32e3d7d/src/meta.ts#L381) but I will add an input to allow to change the name of this target.

Reply:
Done